src/components/ProductDetail.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+function ProductDetail() {
```

Comment:
the product detail component is no different than how you coded it in homework 1.
don't be afraid to look back at previous projects to help you out.